packaging/fedora/snapd.spec

```diff
@@ -577,8 +587,12 @@ pushd ./cmd
 %make_install
 # Undo the 0000 permissions, they are restored in the files section
 chmod 0755 %{buildroot}%{_sharedstatedir}/snapd/void
+%if %{with apparmor}
+install -m 644 -D snap-confine/snap-confine.apparmor %{buildroot}%{_sysconfdir}/apparmor.d/usr.libexec.snapd.snap-confine
```

Comment:
+1 though we could do some work to make `libexec` somehow computed from `%{_libexecdir}`